src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -152,12 +152,37 @@ def translate_outputs(isce_output_dir: Path, product_name: str):
     cmd = (
         'gdal_calc.py '
         f'--outfile {product_name}/{product_name}_{wrapped_phase.suffix}.tif '
-        f'-A {isce_output_dir / wrapped_phase.name} '
+        f'-A {isce_output_dir / wrapped_phase.name} --A_band={wrapped_phase.band} '
         '--calc angle(A) --type Float32 --format GTiff --NoDataValue=0 '
         '--creation-option TILED=YES --creation-option COMPRESS=LZW --creation-option NUM_THREADS=ALL_CPUS'
     )
     subprocess.check_call(cmd.split(' '))
 
+    ds = gdal.Open(str(isce_output_dir / 'los.rdr.geo'), gdal.GA_Update)
+    ds.GetRasterBand(1).SetNoDataValue(0)
+    ds.GetRasterBand(2).SetNoDataValue(0)
+    del ds
+
+    incidence_angle = ISCE2Dataset('los.rdr.geo', 'lv_theta', 1)
+    cmd = (
```

Comment:
From https://github.com/insarlab/MintPy/blob/df96e0b73f13cc7e2b6bfa57d380963f140e3159/src/mintpy/objects/stackDict.py#L732-L737:

```python3
data = 90. - (data * 180. / np.pi)              # hyp3/gamma to mintpy/isce2 convention
```

I confirmed that this is the inverse of the expression used in the `gdal_calc` command here. That is, if the first expression is $f(A)$ and the second expression is $g(A)$, then $f(g(A)) = g(f(A)) = A$.

It may be worth leaving a comment:

```suggestion
    # Performs the inverse of the operation performed by MintPy:
    # https://github.com/insarlab/MintPy/blob/df96e0b73f13cc7e2b6bfa57d380963f140e3159/src/mintpy/objects/stackDict.py#L732-L737
    cmd = (
```